api/client/trust.go

```diff
@@ -37,7 +37,10 @@ import (
 	"github.com/docker/notary/tuf/data"
 )
 
-var untrusted bool
+var (
+	releasesRole = data.CanonicalTargetsRole + "/releases"
```

Comment:
Ha! I commented just the opposite :-D https://github.com/docker/docker/pull/18887#discussion_r48383924